lib/modules/manager/file-match.ts

```diff
@@ -0,0 +1,7 @@
+export function fileMatch(...args: RegExp[]): string[] {
+  return args.map((arg) => {
+    let { source: result } = arg;
+    result = result.replace(/\((?!:\?)/g, '(?:');
```

Comment:
I'm not sure about automatic non-capturing groups, though.
If this is a good idea, then I need to extend this regex in order to exclude any other special grouping variations.

Reply:
Why does capturing matter?

Reply:
Non-capturing regexes should be a bit more performant, though my primary goal here was to enable IDE regex highlight.